src/components/tooltip/Tooltip.tsx

```diff
@@ -279,36 +279,40 @@ class Tooltip extends React.Component<TooltipProps, State> {
             targetAttachment: tetherPosition.targetAttachment,
         };
 
-        const tooltip = (
-            <>
+        const renderTooltip = () => (
+            <div className={classes} id={this.tooltipID} role="tooltip" aria-live="polite">
                 {text}
                 {withCloseButton && (
                     <PlainButton className="tooltip-close-button" onClick={this.closeTooltip}>
                         <IconClose className="bdl-Tooltip-iconClose" width={14} height={14} />
                     </PlainButton>
                 )}
-            </>
+            </div>
         );
 
-        const tooltipWithStopBubble = (
+        const renderTooltipWithStopBubble = () => (
             <div
+                className={classes}
                 role="presentation"
                 onClick={this.handleTooltipEvent}
                 onContextMenu={this.handleTooltipEvent}
                 onKeyPress={this.handleTooltipEvent}
             >
-                {tooltip}
+                <div id={this.tooltipID} role="tooltip" aria-live="polite">
+                    {text}
```

Comment:
can we pull out this part into a shared component that's rendered whether it's with or without stop-bubble? and then all that changes is the div that wraps it?

Reply:
this part being the `{text}` and the `{withCloseButton && ...` part

Reply:
ok